sdcm/cluster_aws.py

```diff
@@ -479,13 +479,21 @@ def get_console_output(self):
         output = self._ec2_service.meta.client.get_console_output(
             InstanceId=self._instance.id,
         )
-        return output['Output']
+        if 'Output' in output:
+            return output['Output']
+        else:
+            self.log.info('Some error during getting console output')
```

Comment:
also maybe those print should be warning level ?

Reply:
Agree, warning is better level. Fix